### ISSUE TITLE
Stroke based svg not working with Vivus

### ISSUE THREAD

**Issue by youriwims** [maxwellito/vivus#187]
[HomeDrawing.6.svg.zip](https://github.com/maxwellito/vivus/files/1799734/HomeDrawing.6.svg.zip)

**Vivus version**: _Latest (tested with Vivus Instant)_

**Browser**: _Chrome_

**Steps to reproduce it**: _I created my svg with Inkscape and made sure to have no fill in my text, only the stroke. I also converted the object to a path and uploaded this into the tester, Vivus Instant. For some reason, it's not working.I've attached my svg and would really appreciate any help. I'm a complete newbie to GitHub and SVGs in general, so any assistance would be appreciated.Thanks!_

**Comment by youriwims** [comment]
*Note: I am not receiving an error message, the animation just isn't happening.

**Comment by maxwellito** [comment]
Hi Youri,

No worries, you did nothing wrong. Even better you converted your text into outlines, which is a good point. The problem was due to the output formatting of Inkscape and missing checks in my code. I discovered this problem yesterday by an issue created by @kdhooper.

To make it short, Inkscape output SVG with `strokeDasharray` and `strokeDashoffset` style properties set even when it's not necessary. Vivus-instant use CSS classes to animate paths, but CSS classes are a lower priority to inline CSS. So vivus-instant was powerless. 

This problem is now fixed and deployed. I just tested with your SVG and it works like a charm.

PS: Welcome to GitHub and the open source world!
